docs/kytConfig.md

```diff
@@ -37,6 +37,8 @@ modifyWebpackConfig: (baseConfig, options) => {
 }
 ```
 
+> **NOTE:** [babel-polyfill](https://babeljs.io/docs/usage/polyfill/) is included in the Webpack configs' entry.main arrays by default. If you use `modifyWebpackConfig()` to override the entry properties and wish to keep using the polyfill, the new entries should go after the polyfill.
```

Comment:
Hi @ccpricenytimes. I just tried the e2e tests locally and I get the same errors without access to the test repo (`NYTimes/kyt-starter-test.git`). 

```
  KYT CLI
    ✓ installs kyt (58433ms)
    ✕ sets up a starter-kyt (3379ms)
    ✕ sets up with the correct files (1ms)
    ✕ sets up the package json scripts (2ms)
    ✕ runs the lint command (749ms)
    ✕ runs the lint-style command (1704ms)
    ✓ runs the tests command (592ms)
    ✕ runs the build command (580ms)
    ✓ starts the app (1076ms)
    ✓ dev (549ms)
    ✓ proto (597ms)
```